src/databricks/labs/remorph/assessments/pipeline.py

```diff
@@ -107,7 +109,9 @@ def _execute_python_step(self, step: Step):
             venv_python = venv_dir / "bin" / "python"
             venv_pip = venv_dir / "bin" / "pip"
 
-            logger.info(f"Creating a virtual environment for Python script execution: ${venv_dir}")
+            logger.info(
+                f"Creating a virtual environment for Python script execution: ${venv_dir} for step: {step.name}"
+            )
```

Comment:
Stray `$`, maybe reword:
```suggestion
            logger.info(f"Creating a virtual environment for Python script execution, step {step.name}: {venv_dir}")
```